examples/firebase-auth-firestore/firebase-fixtures/auth/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "signIn": {
+    "allowDuplicateEmails": false
+  },
+  "usageMode": "DEFAULT"
```

Comment:
this is generated by a Firebase emulators export, when looking for documentation on what usageMode I couldn't find any but found that references to it were removed 12 days ago

https://github.com/firebase/firebase-tools/pull/4607

...I'll remove this the emulators start fine without it